src/ui/components/Icon/styles.scss

```diff
@@ -144,3 +144,7 @@
 .Icon-restart {
   @include icon($name: "restart");
 }
+
+.Icon-caron {
+  @include icon($name: "caron");
```

Comment:
@tofumatt I know it is not a "caron" _per se_ (not even sure what a caron actually is) but I was searching for "inverted caret" and found this word 😝

Reply:
Oh, hah, question answered.

I would rather just call it `inverted-caret`. If you had to google what it was called I'd say it's not obvious 😄